src/components/AutoGrid/index.tsx

```diff
@@ -16,7 +16,7 @@ const AutoGrid: React.FC<Props> = ({ gap = 1, columns = [1, 5], ...props }) => (
       mq({
         display: "grid",
         width: "100%",
-        gridGap: gap && theme.spacing[gap],
```

Comment:
What's the rationale behind this name change?

Reply:
The default key to use the array values for space in onno is spacing. You can modify it and use your own name, but we decided to use the default naming convention onno uses.